Gemfile.lock

```diff
@@ -10,4 +10,4 @@ DEPENDENCIES
   asciidoctor (~> 2.0, >= 2.0.10)
 
 BUNDLED WITH
-   1.17.2
+   1.17.2
```

Comment:
I assume this was added automatically? Strange one.

Reply:
Probably yes.